src/sdk.js

```diff
@@ -122,11 +126,20 @@ const authenticateInterceptors = [
 const loginInterceptors = [
   new AddClientIdToParams(),
   new AddGrantTypeToParams(),
+  new AddIsLoggedInAsToContextFromParams(),
```

Comment:
Is your plan to keep this until re removed the deprecated support for `code` in `login`? If we keep this, then we are still coupling "login as" to the method "authorization code grant type".

Reply:
Yes, that's the plan. I don't think we can do anything else right now without breaking. And yes it's still coupling login as to authorization grant type but only in the deprecated part. We can remove this, and do the breaking change, later in the future if we (ever) introduce another login flow with authorization code grant type.